static/css/components/header-bar.less

```diff
@@ -228,6 +229,7 @@
     -webkit-box-flex: 1;
     -ms-flex: 1;
     flex: 1;
+    z-index: 1;
```

Comment:
Please use the appropriate variable in static/css/less/z-index.less - it helps us keep these under control.

Reply:
Done.